app/editor/src/features/admin/event-of-the-day/EventOfTheDayList.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { toast } from 'react-toastify';
+import { useContent, useLookup } from 'store/hooks';
+import { useFolders } from 'store/hooks/admin';
+import { FlexboxTable, FormPage, IFolderContentModel, Settings } from 'tno-core';
+
+import { useColumns } from './hooks';
+import * as styled from './styled';
+
+/**
+ * Provides a list of all topics.
+ * Provides CRUD form for topics.
+ * @returns Component
+ */
+const EventOfTheDayList: React.FC = () => {
+  const [{ settings }] = useLookup();
+
+  const [, { getContentInFolder }] = useFolders();
+  const [, { updateContentTopics }] = useContent();
+
+  const [loading, setLoading] = React.useState(false);
+  const [items, setItems] = React.useState<IFolderContentModel[]>([]);
+
+  React.useEffect(() => {
+    if (!items.length && !loading) {
+      const eventOfTheDayFolderId = settings.find(
+        (s) => s.name === Settings.EventOfTheDayFolder,
+      )?.value;
+      if (eventOfTheDayFolderId) {
+        setLoading(true);
+        getContentInFolder(+eventOfTheDayFolderId)
+          .then((data) => {
+            setItems(data);
+          })
+          .catch(() => {})
+          .finally(() => {
+            setLoading(false);
+          });
+      } else {
+        toast.error(`${Settings.EventOfTheDayFolder} setting needs to be configured.`);
+      }
+    }
+  }, [settings, getContentInFolder, items.length, loading]);
```

Comment:
Ideally the dependency would use `eventOfTheDayFolderId` instead of all `settings`.  You can get a lot of re-renders this way.